phpBB/includes/acp/acp_profile.php

```diff
@@ -757,6 +757,10 @@ function main($id, $mode)
 				$s_one_need_edit = true;
 			}
 
+			if (!isset($this->type_collection[$row['field_type']]))
+			{
+				continue;
+			}
```

Comment:
These changes are not in the diff, it has been part of the merge conflict resolution